src/actions/pois.action.js

```diff
@@ -80,6 +84,10 @@ export function setSelectedPOI(poi) {
   return dispatch => dispatch(poiSelected(poi))
 }
 
+export function setPreviewedPOI(poi) {

```

Comment:
and for this function it might sound a little better to call it `setPreviewingPOI`. i know its a little inconsistent with `setSelectedPOI` but i think thats just english being inconsistent with how things sound